# SBG Ellipse Messages
# Submessage

# True if the time stamp field represents a delay instead of an absolute time stamp.
bool is_delay_time

# True if the pressure field is filled and valid.
bool pressure_valid

# True if the barometric altitude field is filled and valid.
bool altitude_valid

# True if the differential pressure field is filled and valid.
bool pressure_diff_valid

# True if the true airspeed field is filled and valid.
bool air_speed_valid

# True if the output air temperature field is filled and valid.
bool air_temperature_valid